mldsa/native/aarch64/src/poly_caddq_asm.S

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) The mldsa-native project authors
+ * SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+ */
+#include "../../../common.h"
+
+#if defined(MLD_ARITH_BACKEND_AARCH64) && !defined(MLD_CONFIG_MULTILEVEL_NO_SHARED)
+
+.macro caddq inout
+        sshr tmp.4s, \inout\().4s, #31
+        mls \inout\().4s, tmp.4s, q_reg.4s
```

Comment:
Nit: `shr` + `mla` would be a bit more intuitive?

Reply:
Yes, good point. Changed.